home/models.py

```diff
@@ -83,11 +83,16 @@ class Question(models.Model):
     is_edited = models.BooleanField(default=False)
 
     class Meta:
-        ordering = ['-publish_date']  # ordered by publish_date
+        ordering = ['-publish_date']  # default ordered by publish_date
 
     def __str__(self):
         return f"Question #{self.id} : {self.title} ( {self.publish_date.strftime('%d/%m/%Y %H:%M')} )"
 
+    @classmethod
```

Comment:
If you need a method that returns an information that i relevant for a particular questions oyu can adde a regular method instead of a class method, that way you can avoid the extra DB query. The same comment seem to be relevant to all the methods you've added here.

Reply:
> You need to rebase this PR - it has conflicts - please don't leave it up to me to tell you this, you can look at the PR screen and scroll down just like I can.

Sorry I didn't notice the new PR install Pytest was merged.
I fixed my PR and is ready to review